Dockerfile.template

```diff
@@ -85,10 +86,28 @@ RUN sudo apt-get update && sudo apt-get install -y \
 	php -m | grep "gd" && \
 	php -m | grep "sockets"
 
+
+	# Copies the docker-php-ext helper scripts and allows them to be used in the image
+COPY /docker-php-scripts/docker-php-* /usr/local/bin/
+COPY /docker-php-scripts/install-php-extensions /usr/local/bin/	
+
+RUN cd /usr/local/bin && \
+	sudo chmod +x /usr/local/bin/install-php-extensions /usr/local/bin/docker-php-ext-configure /usr/local/bin/docker-php-ext-enable /usr/local/bin/docker-php-ext-install /usr/local/bin/docker-php-source
```

Comment:
These two lines can be simplified.

1. The `cd` isn't needed.
2. The second line can be `sudo chmod +x /docker-php-scripts/docker-php-* /usr/local/bin/install-php-extensions